tests/phpunit/api/v4/Action/BasicCustomFieldTest.php

```diff
@@ -47,39 +47,45 @@ public function testWithSingleField() {
       ->addValue('data_type', 'String')
       ->execute();
 
+    // Individual fields should show up when contact_type = null|Individual but not other contact types
+    $getFields = Contact::getFields()->setCheckPermissions(FALSE);
+    $this->assertContains('MyIndividualFields.FavColor', $getFields->execute()->column('name'));
+    $this->assertContains('MyIndividualFields.FavColor', $getFields->setValues(['contact_type' => 'Individual'])->execute()->column('name'));
+    $this->assertNotContains('MyIndividualFields.FavColor', $getFields->setValues(['contact_type' => 'Household'])->execute()->column('name'));
```

Comment:
@seamuslee001 this test asserts that fields for Individual are returned even if no contact type is specified.

Reply:
@colemanw i was more thinking of if you specified Individual in the contact type when doing a getfields that custom fields for All contacts are still returned

Reply:
Yes it does.